### PR TITLE
Update siri-shortcuts.md

### PR DESCRIPTION
**Pull request by petereit**
Corrections and clarifications in steps 2-4, renumbered

### CODE REVIEW

**Comment by homeassistant** [comment]
Hi @petereit, 

It seems you haven't yet signed a CLA. Please do so [here](https://home-assistant.io/developers/cla_sign_start/?pr=home-assistant&#x2F;companion.home-assistant#375).

Once you do that we will be able to review and accept this pull request.

Thanks!

File: docs/integrations/siri-shortcuts.md [hunk 1]
```diff
@@ -8,16 +8,17 @@ With iOS 13 or later and the Home Assistant Companion App, you can take advantag
...
-6. Scroll through the list of available services and find `light.turn_on`.
-7. As long as the Dictionary action is above the Home Assistant action there is no need to enter anymore details. If you prefer not to use the Dictionary action, you can select "Show More" and enter the service data in JSON format in the "Service Data" field
-8. Tap next and enter or record a name/phrase to use with "Hey, Siri" to trigger the shortcut.
+2. Tap the Plus icon at the top right to create a new shortcut
+3. Tap Add Action and add a "Dictionary" item.
+4. In the Dictionary item, tap "Add new item", tap "Text" then add `entity_id` as the key and `light.porch` as the text
+5. Tap the large plus to add another action, search for "Home Assistant" and select "Call Service".
+6. Tap "Service" which is highlighted in "Call Service with data"
+7. Scroll through the list of available services and find `light.turn_on`.
+8. As long as the Dictionary action is above the Home Assistant action there is no need to enter anymore details. If you prefer not to use the Dictionary action, you can select "Show More" and enter the service data in JSON format in the "Service Data" field
```
Comment (TomBrien) [inline docs/integrations/siri-shortcuts.md:20]:
```suggestion
8. As long as the Dictionary action is above the Home Assistant action there is no need to enter anymore details. If you prefer not to use the Dictionary action, you can select "Show More" and enter the service data in JSON format in the "Service Data" field.
```

File: docs/integrations/siri-shortcuts.md [hunk 2]
```diff
@@ -8,16 +8,17 @@ With iOS 13 or later and the Home Assistant Companion App, you can take advantag
...
-4. Tap the large plus to add another action, search for "Home Assistant" and select "Call Service".
-5. Tap "Service" which is highlighted in "Call Service with data"
-6. Scroll through the list of available services and find `light.turn_on`.
-7. As long as the Dictionary action is above the Home Assistant action there is no need to enter anymore details. If you prefer not to use the Dictionary action, you can select "Show More" and enter the service data in JSON format in the "Service Data" field
-8. Tap next and enter or record a name/phrase to use with "Hey, Siri" to trigger the shortcut.
+2. Tap the Plus icon at the top right to create a new shortcut
+3. Tap Add Action and add a "Dictionary" item.
+4. In the Dictionary item, tap "Add new item", tap "Text" then add `entity_id` as the key and `light.porch` as the text
+5. Tap the large plus to add another action, search for "Home Assistant" and select "Call Service".
+6. Tap "Service" which is highlighted in "Call Service with data"
```
Comment (TomBrien) [inline docs/integrations/siri-shortcuts.md:18]:
```suggestion
6. Tap "Service" which is highlighted in "Call Service with data".
```

File: docs/integrations/siri-shortcuts.md [hunk 3]
```diff
@@ -8,16 +8,17 @@ With iOS 13 or later and the Home Assistant Companion App, you can take advantag
...
 
 1. Open the Shortcuts app (included by default with iOS, can be re-installed from the [App Store](https://apps.apple.com/us/app/shortcuts/id915249334) if you deleted it)
-2. Tap "Add Item" and search for "Dictionary"
-3. Add a "Text" item and add `entity_id` as the key and `light.porch` as the text
-4. Tap the large plus to add another action, search for "Home Assistant" and select "Call Service".
-5. Tap "Service" which is highlighted in "Call Service with data"
-6. Scroll through the list of available services and find `light.turn_on`.
-7. As long as the Dictionary action is above the Home Assistant action there is no need to enter anymore details. If you prefer not to use the Dictionary action, you can select "Show More" and enter the service data in JSON format in the "Service Data" field
-8. Tap next and enter or record a name/phrase to use with "Hey, Siri" to trigger the shortcut.
+2. Tap the Plus icon at the top right to create a new shortcut
```
Comment (TomBrien) [inline docs/integrations/siri-shortcuts.md:14]:
```suggestion
2. Tap the Plus icon at the top right to create a new shortcut.
```

File: docs/integrations/siri-shortcuts.md [hunk 4]
```diff
@@ -8,16 +8,17 @@ With iOS 13 or later and the Home Assistant Companion App, you can take advantag
...
+3. Tap Add Action and add a "Dictionary" item.
+4. In the Dictionary item, tap "Add new item", tap "Text" then add `entity_id` as the key and `light.porch` as the text
```
Comment (TomBrien) [inline docs/integrations/siri-shortcuts.md:16]:
```suggestion
4. In the Dictionary item, tap "Add new item", tap "Text" then add `entity_id` as the key and `light.porch` as the text.
```

**Review by TomBrien — CHANGES_REQUESTED** [review]
This is a huge improvement. Thanks very much. Just some tiny changes that are from the original any way. Thanks again :)

**Review by TomBrien — APPROVED** [review]

**Pull request merged**